js/widgets/forms/select.js

```diff
@@ -95,49 +94,55 @@ return $.widget( "mobile.selectmenu", $.extend( {
 			classes = " ui-button-right";
 		}
 
-		if ( inline ) {
-			classes += " ui-button-inline";
-		}
-		if ( mini ) {
-			classes += " ui-mini";
-		}
+		this._removeClass( this.element, null, "ui-button-left ui-button-right" );
+		this.element = this.element;
+		this.selectWrapper = $( "<div>" );
+		this._addClass( this.selectWrapper, "ui-selectmenu", classes );
+		this.selectWrapper.insertBefore( this.element );
+		this.element.detach();
 
-		this.select = this.element.removeClass( "ui-button-left ui-button-right" ).wrap( "<div class='ui-select" + classes + "'>" );
-		this.selectId = this.select.attr( "id" ) || ( "select-" + this.uuid );
+		this.selectId = this.element.attr( "id" ) || ( "select-" + this.uuid );
 		this.buttonId = this.selectId + "-button";
-		this.label = $( "label[for='" + $.mobile.path.hashToSelector( this.selectId ) + "']" );
-		this.isMultiple = this.select[ 0 ].multiple;
+		this.isMultiple = this.element[ 0 ].multiple;
+
+		this.element.appendTo( this.selectWrapper );
+		this.label = this.element.labels().first();
 	},
 
 	_destroy: function() {
-		var wrapper = this.element.parents( ".ui-select" );
-		if ( wrapper.length > 0 ) {
-			if ( wrapper.is( ".ui-button-left, .ui-button-right" ) ) {
-				this.element.addClass( wrapper.hasClass( "ui-button-left" ) ? "ui-button-left" : "ui-button-right" );
+		if ( this.selectWrapper.length > 0 ) {
+			if ( this.selectWrapper.is( ".ui-button-left, .ui-button-right" ) ) {
+				this._addClass( null,
```

Comment:
... and if you leave out the element to which you wish to apply the classes, it becomes ```this.element```, so the classes are in the correct parameter, AFAICT.